components/Layout.js

```diff
@@ -1,10 +1,12 @@
 import Head from "next/head"; // HTML head
 import Link from "next/link"; // Routing
-import eth from "@state/eth"; // ETH state container
+import Modal from "@components/Modal"; // Component: Modal
+import vechain from "@state/vechain"; // Vechain state container
 import governance from "@state/governance"; // Governance state container
 import NextNProgress from "nextjs-progressbar"; // Navigation progress bar
 import styles from "@styles/components/Layout.module.scss"; // Component styles
 import Jazzicon, { jsNumberForAddress } from "react-jazzicon"; // Address -> Avatar
+import {useState} from "react";
```

Comment:
Maintain the local style (also happens to align with Proxima's but that's a side point):
```suggestion
import { useState } from "react";
```